discord.photography/Portfolios.html

```diff
@@ -21,14 +21,13 @@
 
 	<div class="main">
 		<h4>
-
-			<li><a href="index.html">Home &nbsp;</a></li>
-
-			<li><a href="Updates/Versions.html">Updates &nbsp;</a></li>
-			
-			<li><a href="Portfolios.html">Portfolios&nbsp;</a></li>
-			
-			<li><a href="Staff.html">Staff</a></li>
+			<a href="index.html">Home &nbsp;</a>
+	
+			<a href="Versions.html">Versions &nbsp;</a>
+	
+			<a href="Portfolios.html">Portfolios&nbsp;</a>
+	
+			<a href="Staff.html">Staff</a>
```

Comment:
We should not use `&nbsp;` for a navigation that needs spacing inbetween. Rather should use margin between the elements or use flex on the container with `space-between`